examples/client/Locomotion/src/pages/Profile/Phone.js

```diff
@@ -49,10 +47,9 @@ const Phone = ({ navigation }) => {
   }, []);
 
 
-  const onVerifyCaptcha = async (token) => {
-    Mixpanel.setEvent('Captcha Verified successfully', { token });
```

Comment:
token is a reserved word in mixpanel, caused this log to never be sent
https://docs.mixpanel.com/docs/data-structure/events-and-properties

Reply:
what are you doing awake on 7 am?

Reply:
oww edgy

that's why we need typescript everywhere

Reply:
Dreaming of captcha issues